CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -66,13 +66,15 @@ private bool IsDeveloper(JToken user)
 
         public IList<TimeSheet> RetrieveTimeSheets(DateTimeOffset dateFrom, DateTimeOffset dateTo)
         {
-            var address = $"/api/v2/time_entries?from={ToHarvestApiString(dateFrom)}&to={ToHarvestApiString(dateTo)}";
-            
-            var response = GetApiResponse(address);
-            
-            response.Wait();
+            var apiResponse = RetrieveATimeSheet(dateFrom, dateTo, 1);
+
+            var totalPages = (int) apiResponse["total_pages"];
+
+            for (var page = 2; page <= totalPages ; page++)
```

Comment:
Are we assuming we will never get more than 2 pages of time sheet entries?

Reply:
Not sure how well this would handle getting only one page back?

Reply:
I think total_pages takes any number of pages. page starts at 2 because the first page has already been retrieved above.

Reply:
I think because we already set it up for the first page, the for loop starts with the second page. The number of pages depends on the totalPages.

Reply:
In regards to one page results what would `totalPages` be set to?

Reply:
This makes sense. I think more tests are needed for clarity.

Reply:
page <= totalPages  if its only 1 page then (2 <= 1) if false, therefore the loop wont execute

Reply:
Ok makes sense. 👍

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/33/commits/7a14902931aef968d2d0de1e54922358413c56b5. Added another unit test for handling time sheet entries which only have one page.